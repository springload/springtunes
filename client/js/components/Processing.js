import React from 'react';

const Processing = () => (
  <div className='loading'>
      <img src={ '../img/processing.gif' } alt='Processing... please wait...' width='24' height='24' />
  </div>
);

export default Processing;
