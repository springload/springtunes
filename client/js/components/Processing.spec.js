import React from 'react';
import Processing from './Processing';
import ReactTestUtils from 'react-addons-test-utils';

describe('Processing component', () => {
    it('renders correctly with min params', () => {
        const shallowRenderer = ReactTestUtils.createRenderer();
        const result = shallowRenderer.render(
            <Processing />
        );
        expect(result).toMatchSnapshot();
    });
});
