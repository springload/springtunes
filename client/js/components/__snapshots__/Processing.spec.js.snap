exports[`Processing component renders correctly with min params 1`] = `
<div
  className="loading">
  <img
    alt="Processing... please wait..."
    height="24"
    src="../img/processing.gif"
    width="24" />
</div>
`;
